include/ocpp/common/websocket/websocket_base.hpp

```diff
@@ -36,6 +36,7 @@ struct WebsocketConnectionOptions {
     std::optional<std::string> hostName;
     bool verify_csms_common_name;
     bool use_tpm_tls;
+    std::optional<std::string> iface_or_ip;    ///< The interface of the connection or the ip address of the interface
```

Comment:
Looks good, however was the equivalent config value added to the db schema?

Reply:
What do you mean?

This will not be a value from the device model but it is returned in the future of the configure_network_connection_profile_callback function

Reply:
Understood, I missed that part.